let ValidationError;_74d‍.x([["default",()=>_74d‍.o]]);_74d‍.w("joi",[["ValidationError",["ValidationError"],function(v){ValidationError=v}]]);

const errorHandler = (err, req, res, next) => {

    let statusCode = 500;

    let data = {
        message: 'Internal Server Error',
        originalError: err.message
    }

    if(err instanceof ValidationError) {
        statusCode = 422;
        data = {
            message: err.message,
        };
    }
    
    return res.status(statusCode).json(data);

}

_74d‍.d(errorHandler);